### ISSUE TITLE
Document Docker Image Stack for Larger Drupal Local Dev Initiative

### ISSUE THREAD

**Issue by rickmanelius** [drud/ddev#555]
## What happened (or feature request):

TL;DR We're looking for places that we could potentially standardize alongside the other Drupal local dev tools as part of a convo that happened during BADCamp.

More information
* [Meeting notes 2017-11-08](https://docs.google.com/document/d/13jwooqfRa-5yuoWpswmlb4cY4Y64OcW5gh4keFNeIcI/edit#)
* [Docker Stack for Drupal and Beyond Sheet](https://docs.google.com/spreadsheets/d/1JaUag8beeQf7fyXIqeznvYdDwt0tsDM0ySks6NwNtZM/edit#gid=2089540011)

This is a prerequisite for moving forward in this conversation and I keep becoming bottlenecked. If there is anyone else on the team that can take this on, I think it would be a valuable exercise nonetheless.

**Comment by rickmanelius** [comment]
@rfay @cweagans In conjunction with doing this for the local dev conversations, we had an internal discussion on identifying any notable mismatches with our hosting stack. If either of you is willing to take a stab at this, it keeps slipping down my priorities because I'm a bit slow on Docker compose and querying details from Docker images/upstream images.

**Comment by rickmanelius** [comment]
This was done and those involved were emailed regarding its completion.
